var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -196,15 +196,18 @@ class Openmpi(AutotoolsPackage):
     variant('thread_multiple', default=False,
             description='Enable MPI_THREAD_MULTIPLE support')
     variant('cuda', default=False, description='Enable CUDA support')
+    variant('hwloc', default='internal' , values=('internal', 'external'),
```

Comment:
for consistency with other packages (see `hypre`), would you mind calling this `internal-hwloc` and make it boolean with default=False (as this is the previous behaviour)?